plugins/event/sqs/sqs.go

```diff
@@ -0,0 +1,248 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package sqs
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"sync"
+
+	"github.com/aws/aws-sdk-go/service/sqs"
```

Comment:
Given this is a new integration, is it possible to use the latest v2 SDK?

Reply:
I tried that initially, but it conflicts with the feedback to use `awsutil` for consistent AWS authentication.